internal/credit/feature.go

```diff
@@ -28,6 +28,26 @@ func (e *FeatureNotFoundError) Error() string {
 	return fmt.Sprintf("feature not found: %s", e.ID)
 }
 
+type FeatureInvalidFiltersError struct {
+	RequestedFilters    map[string]string
+	MeterGroupByColumns []string
+}
+
+func (e *FeatureInvalidFiltersError) Error() string {
+	return fmt.Sprintf("invalid filters for feature: %v, available columns: %v", e.RequestedFilters, e.MeterGroupByColumns)
+}
+
+type FeatureWithNameAlreadyExistsError struct {
+	Name string
+	ID   FeatureID
+}
+
+func (e *FeatureWithNameAlreadyExistsError) Error() string {
+	// Is it an issue that we leak ID on another Feature here?
+	// Shouldn't be an isue as it's namespaced.
+	return fmt.Sprintf("Feature %s with name %s already exists.", e.ID, e.Name)
```

Comment:
Usually go errors are starting with lowercase and don't have ending dots. 

Given that this is the API error we can use this format, but then IMHO we should be consistent with the other error messages (e.g. we should change those too).

Reply:
changed to lowercase non-sentence